test/core/Graphics.js

```diff
@@ -128,4 +128,33 @@ describe('PIXI.Graphics', () =>
             expect(graphics.containsPoint(point)).to.be.false;
         });
     });
+
+    describe('complex drawing #1: draw triangle, rounder rect and an arc (issue #3433)', () =>
```

Comment:
`describe` should be the API, since this is a test for the change you made, would suggest this be `arc`. Please include you're current description as an inline-comment.